topics/assembly/tutorials/vgp_workflow_training/tutorial.md

```diff
@@ -178,58 +178,7 @@ Once we have imported the datasets, the next step is to import the workflows nec
-> | Scaffolding with Hi-C workflow (WF8) | [https://raw.githubusercontent.com/iwc-workflows/Scaffolding-HiC-VGP8/v0.2/Scaffolding-HiC-VGP8.ga](https://raw.githubusercontent.com/iwc-workflows/Scaffolding-HiC-VGP8/v0.2/Scaffolding-HiC-VGP8.ga)|
->
-> 1. Copy the workflow URL into clipboard
->
->    1. Right click on a URL in the table above.
->    2. Select "Copy link address" option in the dropdown menu that appears.
->    3. Go to Galaxy
->
->    > <warning-title>Make sure you are logged in!</warning-title>
->    > Ensure that you are logged in into your Galaxy account!
->    {: .warning}
->
-> 2. Import the workflow
->
->    1. Click "Workflow" on top of the Galaxy interface.
->    2. On top-right of the middle pane click "{% icon galaxy-upload %} Import" button.
->    3. Paste the URL you copied into the clipboard at Step 1 above to "Archived Workflow URL" box.
->    4. Click "Import workflow" button.
->
-> This entire procedure is shown in the animated figure below. {% icon warning %} **You need to repeat this process for all four workflows**
->
-> ![Upload via URL](../../images/vgp_assembly/importing_via_url_vgp_specific.png "Importing a workflow via URL.")
->
-{: .hands-on}
-
-> <details-title>Other ways to import workflows the data</details-title>
-> You can import workflows from a variety of different sources including [DockStore](https://dockstore.org), [WorkflowHub](https://workflowhub.eu), or a URL:
->
-> {% snippet faqs/galaxy/workflows_import_from_dockstore.md %}
->
-> {% snippet faqs/galaxy/workflows_import_from_workflowhub.md %}
->
-> {% snippet faqs/galaxy/workflows_import.md %}
->
-{: .details}
-
-Once all four workflows are imported, your workflow list should look like this:
-
-![Workflow menu](../../images/vgp_assembly/imported_workflows.png  "Workflow list. The workflow menu lists all the workflows that have been imported. It provides useful information for organizing the workflows, such as last update and the tags. The workflows can be run by clicking in the play icon, marked in red in the image.")
-
-Once we have imported the datasets and the workflows, we can start with the genome assembly.
+All analyses described in this tutorial are performed using *workflows*--chains of tools--shown in [Fig. 1](#figure-1). Specifically, we will use four workflows corresponding to analysis trajectory **B**: 1, 4, 6, and 8. To use these four workflows you need to import them into your Galaxy account following the steps below. Note: these are not necessarily the latest versions of the actual workflows, but versions that have been tested for this tutorial. To see the latest versions, see the [Galaxy Project VGP workflows page](https://galaxyproject.org/projects/vgp/workflows/) and click on the Dockstore links to import workflows. **Alternatively, for each section of the tutorial, there will be a "Launch [workflow] (View on Dockstore)" link at the beginning, which you can use to launch the workflow.**
```

Comment:
> these are not necessarily the latest versions of the actual workflows

But they are the latest versions: you've linked to IDs like `github.com/iwc-workflows/kmer-profiling-hifi-VGP1/main` which seem to point to the latest version of each workflow. Did you mean to pin these to specific versions instead?

Reply:
ACK thank you for catching that!! 🙏🏼 I'll edit the snippets to pin to specific versions, because that would make more sense to prevent people from using latest versions that might not match up with what's presented in this tutorial

Reply:
ah, I'm sorry, do you happen to know if there is a way to pin the TRS ID to a specific version? clicking to a different version on the [Dockstore page](https://dockstore.org/workflows/github.com/iwc-workflows/kmer-profiling-hifi-VGP1/main:v0.1.5?tab=info) changes just the `Source code` but not the `TRS`... 

I tried changing it to `github.com/iwc-workflows/kmer-profiling-hifi-VGP1/versions/v0.1.5` and `github.com/iwc-workflows/kmer-profiling-hifi-VGP1/v0.1.5` but those lead to a 404 error :(

Reply:
Huh. When i select an old version on dockstores own launch page, and try and launch it in EU it generated an unversioned url: 

https://usegalaxy.eu/workflows/trs_import?trs_server=dockstore.org&trs_id=%23workflow/github.com/iwc-workflows/kmer-profiling-hifi-VGP1/main

that's completely broken useless lol, you select the version once in dockstore and are again presented with a version selector in galaxy? wild. 

i'll poke their API in a bit to see what format is required (if it is possible) to generate more specific urls

Reply:
Ok I've added a version parameter (e.g. `version="v0.1.5"`) to the snippet. Could you please add the versions to each snippet usage? :)

Reply:
success! adding that parameter makes the URL fill like this: `https://usegalaxy.org/workflows/trs_import?trs_server=dockstore.org&run_form=true&trs_id=%23workflow/github.com/iwc-workflows/kmer-profiling-hifi-VGP1/main&trs_version=v0.1.5` and the page that loads automatically loads the wkflw -> wkflw run page. i'll update the snippets to use the version parameter, thank you so much for your help!!

Reply:
for the sake of documentation, here's what happens when the snippet specifies a version # that doesn't exist: the user's taken to this page with the "version xyz doesn't exist" message and then they can pick the version 👍🏼 
<img width="1057" alt="image" src="https://github.com/galaxyproject/training-material/assets/48773001/82ba8ca9-36ac-463f-8f10-aebc4afdebe1">

the tutorial's snippets should now all have a version pinned 📌 🚀